examples/datadog/agent_basic_values.yaml

```diff
@@ -0,0 +1,33 @@
+# Datadog Agent with Logs, APM, Processes, and System Probe enabled
+
+targetSystem: "linux"
+datadog:
+  apiKey: <DATADOG_API_KEY>
```

Comment:
Should we encourage or mention using secrets here instead? Maybe a separate example?

Reply:
👍 i can include `apiKeyExistingSecret` and comment this out